src/deploymentinfo.cpp

```diff
@@ -52,3 +54,49 @@ std::optional<Consensus::BuriedDeployment> GetBuriedDeployment(const std::string
     }
     return std::nullopt;
 }
+
+#define FLAG_NAME(flag) {std::string(#flag), (uint32_t)SCRIPT_VERIFY_##flag},
+const std::map<std::string, uint32_t> g_verify_flag_names{
+    FLAG_NAME(P2SH)
+    FLAG_NAME(STRICTENC)
+    FLAG_NAME(DERSIG)
+    FLAG_NAME(LOW_S)
+    FLAG_NAME(SIGPUSHONLY)
+    FLAG_NAME(MINIMALDATA)
+    FLAG_NAME(NULLDUMMY)
+    FLAG_NAME(DISCOURAGE_UPGRADABLE_NOPS)
+    FLAG_NAME(CLEANSTACK)
+    FLAG_NAME(MINIMALIF)
+    FLAG_NAME(NULLFAIL)
+    FLAG_NAME(CHECKLOCKTIMEVERIFY)
+    FLAG_NAME(CHECKSEQUENCEVERIFY)
+    FLAG_NAME(WITNESS)
+    FLAG_NAME(DISCOURAGE_UPGRADABLE_WITNESS_PROGRAM)
+    FLAG_NAME(WITNESS_PUBKEYTYPE)
+    FLAG_NAME(CONST_SCRIPTCODE)
+    FLAG_NAME(TAPROOT)
+    FLAG_NAME(DISCOURAGE_UPGRADABLE_PUBKEYTYPE)
+    FLAG_NAME(DISCOURAGE_OP_SUCCESS)
+    FLAG_NAME(DISCOURAGE_UPGRADABLE_TAPROOT_VERSION)
+};
+#undef FLAG_NAME
+
+std::string FormatScriptFlags(uint32_t flags)
+{
+    if (flags == 0) return "";
```

Comment:
Thanks for the ping, I re-ran the job. AWS preempted the job and I haven't figured out how to rerun them automatically yet.
Anyone in this org can re-run corecheck jobs by signing in as well!

Reply:
> Is this ever hit? If not, maybe it can be removed, as the remaining code should reach the same conclusion anyway in the rare case that this is hit?

It's hit for block 170060. I think it's worth having it be explicit, as it would also be reasonable to return "NONE" (as in `SCRIPT_VERIFY_NONE`) for that case.